cypress/e2e/payments.cy.js

```diff
@@ -0,0 +1,45 @@
+describe("View a payment", () => {
+    describe("No payments on case", () => {
+        it("displays default message when there are no payments on the case", () => {
+            cy.visit("/payments?id=999");
+            cy.contains("7000-0000-0001");
+            cy.contains("There is currently no fee data available to display.");
+        });
+    });
+
+    describe("Payments on case", () => {
+        beforeEach(() => {
+            cy.visit("/payments?id=800");
+        });
+
+        it("displays payment information if there is a payment on the case", () => {
+            cy.contains("7000-0000-0000");
+            cy.contains("Total paid");
+            cy.contains("£41.00");
+            cy.contains("Fee details");
+            cy.contains("Payments");
+            cy.get('.govuk-details__summary-text').click();
+            cy.get(".govuk-table__body > :nth-child(1) > .govuk-table__header").contains(
```

Comment:
You should be able to simplify these to `cy.contains("Amount");` `cy.contains("£41.00");` etc

Reply:
I thought this might be better to ensure the ordering of info in the table doesn't regress in the future?

Reply:
I think we tend not to test the design or content too much because it can often be changed, whereas if we just assert that the payment data is visible we don't have to update tests when content or design changes